taxcalc/functions.py

```diff
@@ -262,11 +306,11 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900, e19700,
 
     # Casualty
     if e20500 > 0:
-        c37703 = e20500 + ID_Casualty_frt * _posagi
-        c20500 = c37703 - ID_Casualty_frt * _posagi
+        c37703 = e20500 + ID_Casualty_frt * max(0, e00100)
```

Comment:
On Thu, 3 Dec 2015, Sean.Wang wrote:

> 
> In taxcalc/functions.py:
> 
> > @@ -262,11 +306,11 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900, e
> 19700,
> > 
> >      # Casualty
> >      if e20500 > 0:
> > -        c37703 = e20500 + ID_Casualty_frt * _posagi
> > -        c20500 = c37703 - ID_Casualty_frt * _posagi
> > +        c37703 = e20500 + ID_Casualty_frt * max(0, e00100)
> 
> Note _posagi = max(c00100, 0), by adopting an e-code here would definitely improve
> results. But I think we might still want to use calculated c00100 here instead of
> e00100.

Using the E-value gives a better result only for the actual data year 
(which we never simulate anyway) and with no change in the law determining 
AGI (and never gives the correct MTR). We have to use calculated values 
where ever the taxpayer would have calculated them.

dan

> 
> cc @MattHJensen
> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVRWldMJwpzOF-_nS5tlyApz3iopzks5pML5TgaJpZM4Go_up.gif]
> 
> 
>